config/travellers/nerdbabe.yml

```diff
@@ -0,0 +1,96 @@
+name: "nerdbabe"
+profile:
+  web: "http://nerdbabe.de"
+  twitter: "nerdbabe"
+  gravatar: "5a536d00a2ce1f03f2726a83cfdc6ab8"
+hasbeen:
+  cities:
+    - "Aabenraa"
+    - "Aachen"
+    - "Aalborg"
+    - "Aarhus"
+    - "Aberdeen"
+    - "Amsterdam"
+    - "Aschaffenburg"
+    - "Augsburg"
+    - "Bamberg"
+    - "Barcelona"
+    - "Berlin"
+    - {"Bernau": "Bernau, Österreich"}
+    - {"Biarritz": "Biarritz, France"}
+    - "Bochum"
+    - "Bonn"
+    - "Bordeaux"
+    - "Bremen"
+    - "Brighton"
+    - "Broadstairs"
+    - "Brussel"
+    - "Chemnitz"
+    - "Cologne"
+    - "Cuxhaven"
+    - {"Dax": "Dax, France"}
+    - "Dortmund"
+    - "Düsseldorf"
+    - "Den Haag"
+    - "Drammen"
+    - "Dresden"
+    - "Dundee"
+    - "Edinburgh"
+    - "Essen"
+    - "Fredrikstad"
+    - "Frderikshavn"
+    - "Faaborg"
+    - "Firenze"
+    - "Flensburg"
+    - "Fredericia"
+    - "Fehmarn"
+    - "Ghent"
+    - "Glasgow"
+    - "Gelsenkirchen"
+    - "Hamburg"
+    - "Hanover"
+    - "Heidelberg"
+    - "Helgoland"
+    - "Inverness"
```

Comment:
I was hiking there for three weeks. West Highland way, parts of the Great Glen way and then up to the Skye.